src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -933,7 +934,7 @@ private boolean isAccessAuthorized(DataFile df, String apiToken) {
         if (session != null) {
             if (session.getUser() != null) {
                 if (session.getUser().isAuthenticated()) {
-                    user = (AuthenticatedUser) session.getUser();
+                    user = session.getUser();
```

Comment:
I'm not sure, but `user` is a `User`, so even if we cast here, if we actually want to use that information, we would have to cast again anyway.